src/js/edu-benefits/1995/config/form.js

```diff
@@ -138,13 +138,21 @@ const formConfig = {
           initialData: {
           },
           uiSchema: {
+            'view:newService': {
+              'ui:title': 'Do you have any new military service to record?',
+              'ui:options': {
+                hideOnReviewIfFalse: true,
```

Comment:
I think you can remove this. I think it looks weird on the review page if the question is sometimes hidden.